serverlessworkflow/antora.yml

```diff
@@ -110,7 +110,7 @@ asciidoc:
     images_distributions_url: https://kie.apache.org/docs/start/download#container-images
     apple_support_url: https://support.apple.com/guide/mac-help/open-a-mac-app-from-an-unknown-developer-mh40616/mac
     kogito_examples_repository_url: https://github.com/apache/incubator-kie-kogito-examples
-    kogito_operator_repository_rawcontent_url: https://raw.githubusercontent.com/apache/incubator-kie-kogito-serverless-operator
+    kogito_operator_repository_rawcontent_url: https://raw.githubusercontent.com/apache/incubator-kie-tools/refs/heads/{operator_version}/packages/sonataflow-operator
```

Comment:
Is this fine for 10.0.x? Because it seems like operator_version is pointing to "main"
https://github.com/apache/incubator-kie-kogito-docs/blob/6b7bb24786cecdc761c0a98553ccb263f04fbed6/serverlessworkflow/antora.yml#L95

Reply:
Thanks, that is a mistake. Fixed in lastest commit.